segmentstore/server/host/src/main/java/io/pravega/segmentstore/server/host/ServiceStarter.java

```diff
@@ -99,6 +107,17 @@ public void start() throws Exception {
             statsProvider.start();
         }
 
+        log.info("Initializing HealthService ...");
+        healthServiceManager = new HealthServiceManager(serviceConfig.getHealthCheckInterval());
+        healthServiceManager.start();
+
+        log.info("Initializing RESTServer ...");
+        restServer = new RESTServer(serviceConfig.getRestServerConfig(), Set.of(new HealthImpl(
```

Comment:
Just wondering - why do we need a collection here? Isn't it just one element? Is it because that's how the `RESTService` requires a collection? 

FYI `Collections.singleton` is equivalent here.

Reply:
That's correct. `RestApplication` accepts `Set<Object>` in its constructor.